### PR TITLE
Polish News for release 1.0.0

### PR DESCRIPTION
**Pull request by dpseidel**
Reorganized bullets according to tidyverse style guide for news before release. Also polished for consistency throughout.  Updated news/index.html with pkgdown.

### CODE REVIEW

**Comment by dpseidel** [comment]
You'll notice that I added links for the ggplot2 issues linked in news. although not quite kosher with the style guide this is to avoid pkgdown creating spurious links to scales issues automatically.  Easily removed if you prefer.

**Comment by hadley** [comment]
Overall form LGTM - @batpigandme can you please take a proof reading pass too?

File: NEWS.md [hunk 1]
```diff
@@ -1,61 +1,71 @@
...
+  `trim` (@larmarange, #148).
+
+* New `number_format()` provides a generic formatter for numbers (@larmarange, #142).
+  
+* New `pvalue_format()` formats p-values (@larmarange, #145).
 
-* Remove `plyr` and `dichromat` dependencies. `dichromat` is now suggested
- (@dpseidel, #118).  
+* `ordinal_format()` gains new arguments: `prefix`, `suffix`, `big.mark`, `rules`,
+   and rules for French and Spanish are also provided (@larmarange, #149).
```
Comment (batpigandme) [inline NEWS.md]: I'd switch a semi-colon in for the note at the end ⇨ "`rules`; rules for French and Spanish..."

File: NEWS.md [hunk 2]
```diff
@@ -1,61 +1,71 @@
...
+  `trim` (@larmarange, #147).
+  
+* New `time_format()` formats `POSIXt` and `hms` objects (@dpseidel, #88).
 
-* New function `time_format()` formats `POSIXt` and `hms` objects (@dpseidel, #88).
+### Transformations & breaks
 
-* `rescale_mid()` now properly handles NAs (@foo-bar-baz-qux, #104).
+* `boxcox_trans()` is now invertible for `x >= 0` and requires positive values.
+  A new argument `offset` allows specification of both type 1 and type 2 Box-Cox
```
Comment (batpigandme) [inline NEWS.md]: hyphens for "type-1" and "type-2" (since modifying …transformations)

File: NEWS.md [hunk 3]
```diff
@@ -91,12 +101,12 @@
...
+  ([ggplot2-1482](https://www.github.com/tidyverse/ggplot2/issues/1482)).
   
 # scales 0.3.0
 
 * `rescale()` preserves missing values in input when the range of `x` is
-  (effectively) 0 (ggplot2#985).
+  (effectively) 0 ([ggplot2-985](https://www.github.com/tidyverse/ggplot2/issues/985)).
 
 * Continuous colour palettes now use `colour_ramp()` instead of `colorRamp()`.
   This only supports interpolation in Lab colour space, but is hundreds of
```
Comment (batpigandme) [inline NEWS.md]: Is there a word missing here? "is hundreds of…" If no (which is totally possible), toss in a period?

Comment (dpseidel) [reply NEWS.md]: not sure why it's not showing up here but it does continue to say "hundreds of times faster." i've double checked the file.

Comment (batpigandme) [reply NEWS.md]: I believe you! The internet is weird.

**Pull request merged**